packages/react-router/modules/Switch.js

```diff
@@ -51,9 +51,8 @@ class Switch extends React.Component {
   }
 
   render() {
-    const { children } = this.props
+    const { children:routes } = this.props
```

Comment:
It doesn't. But it would *definitely* break under Fiber. Switched it to use React.Children.forEach anyways.